integration-tests/parser/import/test.test.js

```diff
@@ -0,0 +1,10 @@
+const test = require('ava');
+const {exec} = require('../../setup');
+
+test(
+    'parser: able to import variable from module',
```

Comment:
IMHO this is more about desugaring (finding correct module reference for each used value/type) than parser. WDYT?

Reply:
I agree